source/webapp/sites/all/modules/custom/checkbook_custom_breadcrumbs/customclasses/checkbook_custom_breadcrumbs.php

```diff
@@ -81,13 +81,13 @@ public static function getSpendingBreadcrumbTitle()
         $title = NodeSummaryUtil::getInitNodeSummaryTitle($smnid);
       } else {
         $last_id = _getLastRequestParamValue($bottomURL);
-        if ($last_id['vendor'] > 0) {
+        if (isset($last_id['vendor']) > 0) {
```

Comment:
Can you please check if we need check isset($variable) instead of isset($variable) > 0 as IDs might be negative also technically?

Reply:
It is ok to have isset($v) >0.